pkg/ebpf/processor_proctree.go

```diff
@@ -66,10 +69,11 @@ func (t *Tracee) procTreeForkProcessor(event *trace.Event) error {
 	if err != nil {
 		return err
 	}
-	forkFeed.LeaderStartTime, err = parse.ArgVal[uint64](event.Args, "leader_start_time")
+	leaderStartTime, err := parse.ArgVal[time.Time](event.Args, "leader_start_time")
```

Comment:
@geyslan we need to make sure proctree functionality isn't affected with these changes (I remember we had issues with process id hashes not matching after making such changes previosuly). Do we have tests for that?